### PR TITLE
[DEM]Added 2D functionality

### PR DESCRIPTION
**Pull request by farrufat-cimne**
added ComputeConditionRelativeData() to rigidEdge capabilitites.

RigidEdge3D2N - CylinderContinuumParticle2D contact seems to be working as expected now.

### CODE REVIEW

File: applications/DEMApplication/custom_conditions/RigidEdge.cpp [hunk 1]
```diff
@@ -80,7 +80,89 @@ void RigidEdge3D::Initialize() {
...
+        if (points == 4)
+        {
+            Weight[3] = TempWeight[3];
+        }
+        else
+        {
+            Weight[3] = 0.0;
+        }
+    }
+
```
Comment (joaquinirazabal) [inline applications/DEMApplication/custom_conditions/RigidEdge.cpp]: As this is an edge the number, I guess that we can avoid the condition _if (points == 3 || points == 4)_ we will never access it.

File: applications/DEMApplication/custom_conditions/RigidEdge.cpp [hunk 2]
```diff
@@ -80,7 +80,89 @@ void RigidEdge3D::Initialize() {
 }
 
 
+void RigidEdge3D::ComputeConditionRelativeData(int rigid_neighbour_index,
+                                               SphericParticle* const particle,
+                                               double LocalCoordSystem[3][3],
+                                               double& DistPToB,
+                                               array_1d<double, 4>& Weight,
+                                               array_1d<double, 3>& wall_delta_disp_at_contact_point,
+                                               array_1d<double, 3>& wall_velocity_at_contact_point,
```
Comment (joaquinirazabal) [inline applications/DEMApplication/custom_conditions/RigidEdge.cpp]: What do you think about changing the name of these two values?
- wall_delta_disp_at_contact_point -> edge_delta_disp_at_contact_point
- wall_velocity_at_contact_point -> edge_velocity_at_contact_point
Maybe it is not necessary, I do not know...

**Review by joaquinirazabal — COMMENTED** [review]
Great work @farrufat-cimne, this was an important bug in the 2D implementation. I made two minor comments.

**Comment by maceligueta** [comment]
Hi @farrufat-cimne , thanks for adding this feature. Some tests would be great. If possible, add a fast one to the 'tests' folder which is the only one triggered from Travis.

**Comment by farrufat-cimne** [comment]
added small test using 2D DEM continuum particles impacting between them and against a wall.
removed old warnings from previous tests

**Comment by farrufat-cimne** [comment]
@joaquinirazabal I think it can be merged after the checks.

**Comment by joaquinirazabal** [comment]
Great! Thank you, @farrufat-cimne

**Review by joaquinirazabal — DISMISSED** [review]

**Review by maceligueta — DISMISSED** [review]

**Comment by farrufat-cimne** [comment]
fixed typo and updated DEM benchmarks reference values (due to fix in the tangential force calculation from old PR)

**Comment by farrufat-cimne** [comment]
fixed typo in header file. @joaquinirazabal

**Review by maceligueta — APPROVED** [review]

**Comment by loumalouomega** [comment]
Just one comment, test_examples should be moved to Examples repository (https://github.com/KratosMultiphysics/Examples), otherwise this repository becomes huge

**Pull request merged**